### ISSUE TITLE
BUG of @groceristar/groceristar-fetch

### ISSUE THREAD

**Issue by vadim9999** [GroceriStar/react-print-pdf#162]
output log
can't resolve  @groceristar/groceristar-fetch in selector.js.
Bug is in version upper 2.1.10.
On version groceristar-fetch of 2.1.10 working good.

**Comment by atherdon** [comment]
yep it's one of the problems that we have with fetch because it's es5 + I
didn't figure out what is the best configuration of rollup so fetch can be
interpret for all kind of js projects. like nodejs and react at both cases.
another point that Eugene comment a lot of modules at fetch, so we
literally can don't have some of our methods working.

- 1 way to deal with it - just downgrade the fetch to prev version which i
assume you did

- 2 way is to disable fetch/selectors for now, because header don't require
worfking fetch stuff

- 3 way is to try to replace some methods from fetch with plain files from
sd-wrapper

- 4 way is long to try to fix fetch rollup config so it work well. but i
think it's better to finish my code with `projects2.0` branch, where i
replacing/simplifying a lot of methods and move them into es6 style

i'm ok with quickiest way

>
